src/js/components/container/LoginFormContainer.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import Input from "../presentational/Input";
+import InputSubmit from "../presentational/inputSubmit";
+import Buttonify from "../presentational/styled/Buttonify";
+
+class FormContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      username: "",
+      password: ""
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleChange(event) {
+    console.log(this.state)
```

Comment:
Can you add a comment talking about why we have this here.

Reply:
10-2, all fixed!